docker/1.13/Dockerfile.eia

```diff
@@ -1,31 +1,38 @@
 FROM ubuntu:16.04
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
 
+ENV SAGEMAKER_TFS_VERSION="${TFS_SHORT_VERSION}"
+ENV PATH="$PATH:/sagemaker"
+
+ARG PIP=pip3
 ARG TFS_SHORT_VERSION
 
 # nginx + njs
-RUN \
-    apt-get update && \
-    apt-get -y install --no-install-recommends curl gnupg2 && \
-    curl -s http://nginx.org/keys/nginx_signing.key | apt-key add - && \
-    echo 'deb http://nginx.org/packages/ubuntu/ xenial nginx' >> /etc/apt/sources.list && \
-    apt-get update && \
-    apt-get -y install --no-install-recommends nginx nginx-module-njs python3 python3-pip python3-setuptools && \
-    apt-get clean
-
-# cython, falcon, gunicorn, tensorflow-serving
-RUN \
-    pip3 install --no-cache-dir cython falcon gunicorn gevent requests grpcio protobuf && \
-    pip3 install --no-dependencies --no-cache-dir tensorflow-serving-api==1.13.0
+RUN apt-get update \
+ && apt-get -y install --no-install-recommends curl gnupg2 \
+ && curl -s http://nginx.org/keys/nginx_signing.key | apt-key add - \
+ && echo 'deb http://nginx.org/packages/ubuntu/ xenial nginx' >> /etc/apt/sources.list \
+ && apt-get update \
+ && apt-get -y install --no-install-recommends nginx nginx-module-njs python3 python3-pip python3-setuptools \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
 
-COPY ./ /
+# cython, falcon, gunicorn, grpc
+RUN ${PIP} install --no-cache-dir \
+    awscli==1.16.130 \
+    cython==0.29.10 \
+    falcon==2.0.0 \
+    gunicorn==19.9.0 \
+    gevent==1.4.0 \
+    requests==2.21.0 \
+    grpcio==1.24.1 \
+	protobuf==3.10.0
```

Comment:
indentation for protobuf, similar in some other dockerfiles as well

Reply:
done.